doc/lua_api.md

```diff
@@ -2387,6 +2387,48 @@ Table of resulting tool uses:
 
 
 
+Wear Bar Color
+==============
+
+'Wear Bar' is a property of items that defines the coloring
+of the bar that appears under damaged tools.
+If it is absent, the old behavior of green-yellow-red is
+used.
```

Comment:
calling it "default behavior" would make more sense, it's not like we're going to remove it